src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -53,101 +56,37 @@
 /**
  * Controller responsible for "Sell > Catalog > Attributes & Features > Features" page
  */
-class FeatureController extends FrameworkBundleAdminController implements LegacyControllerBridgeInterface, LegacyListControllerBridgeInterface
+class FeatureController extends FrameworkBundleAdminController implements FrameworkBridgeControllerInterface, FrameworkBridgeListControllerInterface
 {
     use AdminControllerTrait;
+    use AdminListControllerTrait;
```

Comment:
Maybe we should rename those traits to match the interfaces: `FrameworkBridgeControllerTrait` and `FrameworkBridheListControllerTrait`